dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityChangeLogService.java

```diff
@@ -93,7 +93,11 @@ public Page<TrackedEntityChangeLog> getTrackedEntityChangeLog(
     }
 
     return jdbcTrackedEntityChangeLogStore.getTrackedEntityChangeLog(
-        trackedEntityUid, trackedEntityAttributes, operationParams.getOrder(), pageParams);
+        trackedEntityUid,
+        trackedEntityAttributes,
+        programUid,
+        operationParams.getOrder(),
+        pageParams);
   }
```

Comment:
We could do it, but then we would have the operations mapper in the store signature, which feels wrong, since the tracker stores always use `QueryParams` instead. 
I didn't do that because it feels a bit of an overkill to have a mapper and `QueryParams` for these two parameters.